.github/workflows/advance-super.yml

```diff
@@ -36,12 +36,15 @@ jobs:
           # We need a token with permission to push.
           token: ${{ secrets.PAT_TOKEN }}
       - uses: ./.github/actions/setup-app
-      - run: sudo apt-get -y install whois
+      # Installing libsecret-1-dev was necessary to get e2e tests to run on ubuntu-24.04 Actions Runners
+      - run: sudo apt-get -y install whois libsecret-1-dev
       - run: yarn workspace superdb-desktop add super@brimdata/super#${{ env.super_ref }}
       - run: yarn workspace superdb-node-client add super@brimdata/super#${{ env.super_ref }}
       - run: yarn lint
       - run: yarn test
       - run: yarn build
+      # The following sysctl setting is to work around https://github.com/brimdata/zui/issues/3194
+      - run: sudo sysctl -w kernel.apparmor_restrict_unprivileged_userns=0
```

Comment:
1. Add this stuff to the "End to end tests" action's run value instead of as new actions.
2. libsecret-1-0 should suffice. The dev package shouldn't be necessary.

Reply:
@nwt: Ok, changes made. The [re-run](https://github.com/brimdata/zui/actions/runs/14628387533/job/41045351466) after making these did complete successfully.